test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -83,6 +83,9 @@
     ('quantized::linear_dynamic', datetime.date(2020, 6, 1)),
     ('quantized::linear_relu', datetime.date(2020, 6, 1)),
     ('quantized::linear', datetime.date(2020, 6, 1)),
+    ('quantized::layer_norm', datetime.date(2020, 6, 30)),
+    ('quantized::group_norm', datetime.date(2020, 6, 30)),
+    ('quantized::instance_norm', datetime.date(2020, 6, 30)),
```

Comment:
I didn't see changes to these functions in the PR, why are they here?

Reply:
fixed, thanks!